ray-curator/ray_curator/backends/base.py

```diff
@@ -70,6 +70,10 @@ def process_batch(self, tasks: list[Task]) -> list[Task]:
 
         # Log performance stats and add to result tasks
         _, stage_perf_stats = self._timer.log_stats()
+        # Consume and attach any custom metrics recorded by the stage during this call
+        custom_metrics = self.stage._consume_custom_metrics()
+        if custom_metrics:
+            stage_perf_stats.custom_stats.update(custom_metrics)
```

Comment:
`custom_metrics` versus `custom_stats` is a bit confusing, could they just be called the same thing?

Reply:
+1